pkg/apiserver/apiserver_test.go

```diff
@@ -201,7 +204,7 @@ func handleLinker(storage map[string]rest.Storage, selfLinker runtime.SelfLinker
 }
 
 func newTestRequestInfoResolver() *RequestInfoResolver {
-	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString("api")}
+	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString()}
```

Comment:
@deads2k I would prefer changing the prefix in the tests to /apis, retaining 'api' as a groupless prefix, and taking a TODO to add a third codec and set of test cases to test the v1-style URLs in this file